target/linux/ramips/dts/rt3352_zte_mf283plus.dts

```diff
@@ -0,0 +1,130 @@
+#include "rt3352.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "zte,mf283plus", "ralink,rt3352-soc";
+	model = "ZTE MF283+";
+
+	aliases {
+		led-boot = &led_wwan_green;
+		led-failsafe = &led_wwan_red;
+		led-running = &led_wwan_blue;
+		led-upgrade = &led_wwan_red;
+		label-mac-device = &wmac;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_wwan_blue: wwan_blue {
+			label = "blue:3g4g";
```

Comment:
Please be consistent here, either wwan or 3g4g ...

Reply:
Good catch, fixed.